doc/examples/monthly-means.ipynb

```diff
@@ -183,7 +108,7 @@
    },
    "outputs": [],
    "source": [
-    "print(ds_weighted)"
+    "ds_weighted"
```

Comment:
would this cell benefit from an update using our new weighted api?

Reply:
Thanks @jhamman!  I considered that, but wasn't quite sure how to work it in with the `groupby` API.  Happy to take suggestions, however.

Reply:
Neither do I to be honest... #3937

Reply:
looks like this will need some engineering so things look good to me for now.